src/auth.d.ts

```diff
@@ -420,6 +420,17 @@ export namespace admin.auth {
      */
     auth_time: number;
 
+    /**
+     * The email corresponding to the user who the ID token belonged to, if available.
+     */
+    email?: string;
+
+    /**
+     * Whether the email corresponding to the user who the ID token belonged to is
```

Comment:
Suggest "Whether or not the email of the user to whom the ID token belonged is verified, provided the user has an email."

Are the valid values true and false?

Reply:
Yes, true or false. Done.